ttnctl/cmd/applications_pf.go

```diff
@@ -33,7 +33,7 @@ function Decoder(bytes) {
 
 		appID := util.GetAppID(ctx)
 
-		conn, manager := util.GetHandlerManager(ctx)
+		conn, manager := util.GetHandlerManager(ctx, appID)
 		defer conn.Close()
 
 		app, err := manager.GetApplication(appID)
```

Comment:
Basically:
```
util.GetHandlerManager(ctx, appID)
```
Gets a handler manager that is preloaded with the correct token.
```
app, err := manager.GetApplication(appID)
```
Fetches the application.